src/laser_utils.cpp

```diff
@@ -125,6 +125,13 @@ karto::LaserRangeFinder * LaserAssistant::makeLaser(const double & mountingYaw)
   }
   node_->get_parameter("max_laser_range", max_laser_range);
 
+  if (max_laser_range <= 0) {
```

Comment:
Is this a real issue you really ran into? This seems like its from a poorly formed laser scan message and it should be fixed by the publisher

Reply:
I just consider it as a parameter a user can set, and it seems it is not related to scan message...